src/dist/replication/test/simple_kv/case-211.act

```diff
@@ -70,14 +70,12 @@ wait:on_task_begin:node=r3,task_code=RPC_GROUP_CHECK
 state:{{r1,pri,2,15},{r2,sec,2,15},{r3,pot,2,0}}
 wait:on_rpc_call:rpc_name=RPC_LEARN,from=r3,to=r1
 inject:on_rpc_reply:rpc_name=RPC_LEARN_ACK,from=r1,to=r3
-state:{{r1,pri,2,15},{r2,sec,2,15},{r3,err,2,0}}
```

Comment:
Specifically, this change is related to commit https://github.com/imzhenyu/rDSN/commit/98760cdb99325be52b92b1d5e0fe42284d99c909 and https://github.com/imzhenyu/rDSN/commit/291d5a05b69f0233b2a47753208d10fc9e3a38fe